Examples/MagicWeather/MagicWeather/Sources/Lifecycle/AppDelegate.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AppDelegate.swift
+//  Magic Weather
+//
+//  Created by Cody Kerns on 12/14/20.
+//
+
+import UIKit
+import Purchases
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        /* Enable debug logs before calling `configure`. */
+        Purchases.debugLogsEnabled = true
+
+        /*
+         Initialize the RevenueCat Purchases SDK.
+         
+            - appUserID is nil, so an anonymous ID will be generated automatically by the Purchases SDK. Read more about Identifying Users here: https://docs.revenuecat.com/docs/user-ids
+         
+            - observerMode is false, so Purchases will automatically handle finishing transactions. Read more about Observer Mode here: https://docs.revenuecat.com/docs/observer-mode
+         */
+        
+        Purchases.configure(withAPIKey: Constants.apiKey,
+                            appUserID: nil,
+                            observerMode: false)
```

Comment:
I think we should set up a purchaserInfoDelegate as well to showcase how to use it, especially now that it's important for the SCA flow